### PR TITLE
Sponsors section prototype

### PR DESCRIPTION
**Pull request by IvanDonat**
I've made a basic layout for the sponsors section. It's far from perfect and could use a lot of fixing but the template is dependent on things like image resolution and such, so I don't know if it's worth the effort of making it look good with these images, just to have to change it afterwards.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -144,6 +145,69 @@ <h1 id="logo"><a href="index.html">GreatUniHack</a></h1>
...
+			<!-- Sponsors -->
+			<section id="sponsors" class="wrapper style1 special fade-up">
+				<div class="container">
+					<header class="major">
+						<h2>Sponsors</h2>
+					</header>
+
+					<!-- For mobile screens, if we want pictures one below another, use 12u(xsmall)
+						If we want 2 pictures per line, use 6u
+						If we want 3 pictures per line, use 3u -->
```
Comment (rignatus) [inline index.html]: Good comments. Consider adding them under the "Sponsors" comment.

File: index.html [hunk 2]
```diff
@@ -50,6 +50,7 @@ <h1 id="logo"><a href="index.html">GreatUniHack</a></h1>
 					<ul>
 						<li><a href="index.html">Home</a></li>
 						<li><a href="#faq" class="scrolly">FAQ</a>
+						<li><a href="#sponsors" class="scrolly">Sponsors</a>
```
Comment (rignatus) [inline index.html]: The list item tag should be closed at the end of each item.

File: index.html [hunk 3]
```diff
@@ -144,6 +145,69 @@ <h1 id="logo"><a href="index.html">GreatUniHack</a></h1>
...
+							</div>
+							<div class="6u 6u(xsmall)">
+								<span class="image fit"><img src="images/webapps.png" alt="" /></span>
+							</div>
+							<div class="12u 12u(xsmall)">
+								<span class="image fit"><img src="images/barclays.png" alt="" /></span>
+							</div>
+						</div>
+					</div>
+
```
Comment (rignatus) [inline index.html]: Electrum sponsor section should be included here.

File: index.html [hunk 4]
```diff
@@ -144,6 +145,69 @@ <h1 id="logo"><a href="index.html">GreatUniHack</a></h1>
...
+								<span class="image fit"><img src="images/oculus.png" alt="" /></span>
+							</div>
+							<div class="1u 1u(xsmall)">
+								<span class="image fit"><img src="images/guh_logo.png" alt="" /></span>
+							</div>
+						</div>
+					</div>
+				</div>
+			</section>
+
```
Comment (rignatus) [inline index.html:193]: Bronze and Partners sponsors section should be included here, in this order.

**Review by rignatus — CHANGES_REQUESTED** [review]
The section looks pretty good.
As the images were only test images, they should not be added. I have left some comments that should be addressed before merging.

**Comment by IvanDonat** [comment]
I've modified the sponsors section a bit, this might be a bit better than the last time. Feedback?
I haven't added all the sponsor types yet as there aren't enough images.

Another note:
Sometimes when we put smaller images we can create an empty div of some size to center them, but then it looks bad on xsmall screens. This could be fixed by creating a 0u in the CSS to be used on xsmall screens, as it doesn't exist atm.

I'm not sure if there is a better way to do it with this layout we've got.

**Comment by IvanDonat** [comment]
Also, maybe if we used icons instead of text for the sponsor type? Or slightly different background shades?

**Comment by andreimuntean** [comment]
Can you please compress the images to less than 20 kB while maintaining their high quality? And please delete the pictures of companies that aren't sponsoring GUH 2017.

I'd suggest closing this pull request and making a new one with complete changes. This is so HackSoc doesn't pull in all the work in progress commits. But up to you.

**Pull request closed without merging**